com.beeper.Beeper.yaml

```diff
@@ -0,0 +1,90 @@
+sdk: org.freedesktop.Sdk
+command: startbeeper
+rename-icon: beeper
+separate-locales: false
+finish-args:
+  # Version required to use the document-portal v4
+  - --require-version=1.7.0
+  # Required due to being a GUI application
+  - --socket=x11
+  # Required to make sure x11 performance is achived on all platforms
+  # At least that's what the legends tell. it might be worth experimenting
+  # with dropping this permission.
+  - --share=ipc
+  # Required to provide Call functionality
+  - --socket=pulseaudio
+  - --device=all
+  # As a chat application, networking is required
+  - --share=network
+  # Required for appindicator in various desktop environments
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # Required to allow screensaver/idle inhibition such as during video calls
+  - --talk-name=org.freedesktop.ScreenSaver
+  # Required for advanced input methods e.g. writing CJK languages
+  - --talk-name=org.freedesktop.portal.Fcitx
+  # Required to store access tokens (persistent logins)
+  - --filesystem=xdg-run/keyring
+  # Required for experimental wayland support
+  - --filesystem=xdg-run/pipewire-0
+  # For correct cursor scaling under Wayland
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /lib/*.la
+  - /lib/*.a
+  - /share/pkgconfig
+  - /share/aclocal
+  - /share/gtk-doc
+  - /share/doc
+  - /share/info
+  - /share/man
+  - /man
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - name: beeper
+    buildsystem: simple
+    build-commands:
+      - chmod +x Beeper.AppImage
+      - ./Beeper.AppImage --appimage-extract
+
+      - desktop-file-edit --set-icon ${FLATPAK_ID} --set-key Exec --set-value 'startbeeper
+        %u' squashfs-root/beeper.desktop
+      - install -D squashfs-root/beeper.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -D startbeeper /app/bin/
+      - install -D com.beeper.Beeper.metainfo.xml /app/share/metainfo/com.beeper.Beeper.metainfo.xml
+      - mkdir -p /app/share/icons/hicolor
+      - cp -r squashfs-root/usr/share/icons/hicolor/* ${FLATPAK_DEST}/share/icons/hicolor
+      # XDG desktop specification doesn't support icons larger than 512x512
+      - rm -rf ${FLATPAK_DEST}/share/icons/hicolor/1024x1024
+      - mv squashfs-root /app/bin/beeper
+      - rm -rf squashfs-root/resources/icons/win
+      - rm Beeper.AppImage
+    sources:
+      - type: script
+        dest-filename: startbeeper
+        commands:
+          - for i in {0..9}; do
+          - test -S $XDG_RUNTIME_DIR/discord-ipc-$i || ln -sf {app/com.discordapp.Discord,$XDG_RUNTIME_DIR}/discord-ipc-$i;
+          - done
+          - export FLATPAK_ID="${FLATPAK_ID:-com.beeper.Beeper}"
+          - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
+          - zypak-wrapper /app/bin/beeper/beeper "$@"
+
+      - type: file
+        only-arches: [x86_64]
+        url:  https://download.beeper.com/versions/3.78.22/linux/appImage/x64
```

Comment:
Yes

Reply:
Well not sure about that, I asked them specifically about it, but I'll see what I can find

Reply:
Unfortunately I can't find anything :c

I have asked them for approval but it's in a private "Beeper help" and they said yes, I believe it might be hard to prove so maybe @cassidyjames who's been talking with the team can give you a more "solid" evidence?

Reply:
What's extra-data?